### PR TITLE
Scaled powerPerSecond with tick duration

### PR DESCRIPTION
**Pull request by Jian-Wuyou**
## Description
Fixed incorrect power per second for tick durations that are not 0.5s

## Changes
Changed `powerPerSecond(int power)` to divide `power` with the tick duration (i.e. if power given to LoreBuilder is 240 and tick duration is 12 ticks, it shows 400 J/s as the final output). This will require that current LoreBuilder values be divided by 2 (i.e. if currently it says `LoreBuilder.powerPerSecond(500)`, it should be changed to `LoreBuilder.powerPerSecond(250)`).

## Related Issues

Resolves: #2835

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by WalshyDev** [comment]
How come you closed this?

**Comment by Jian-Wuyou** [comment]
Just remembered that I still have to rewrite the current LoreBuilder.powerPerSecond calls so that it displays the correct values (i.e. replace the current values with the per-tick usage) otherwise it'll display two times the actual values

**Comment by Jian-Wuyou** [comment]
Example of how it looks like with Lava Generator 2 (20 J/t, tick duration is 0.6s)
image

All energy values are rounded off to 2 decimal places.

File: src/main/java/io/github/thebusybiscuit/slimefun4/utils/LoreBuilder.java [hunk 1]
```diff
@@ -55,12 +57,12 @@ public static String powerBuffer(int power) {
 
     @Nonnull
     public static String powerPerSecond(int power) {
-        return power(power, "/s");
+        return power((float)power * 20f / SlimefunPlugin.getCfg().getInt("URID.custom-ticker-delay"), "/s");
```
Comment (Mooy1) [inline src/main/java/io/github/thebusybiscuit/slimefun4/utils/LoreBuilder.java:60]: should add a static field for 20 / configValue

**Comment by TheBusyBiscuit** [comment]
I am sorry but I have to reject this.
Changing the lore of items will have a ton of unforeseen consequences.
Stacking problems will just be one of them.
This might be possible in the future, but at the moment these changes would cause more problems than they solve.

Your changes would for example result in **every** existing item before these changes no longer being usable in some crafting recipes. So this will likely spawn even more bugs and bug reports than for what it's worth.
Sorry for the effort you put into this, but for now you need to accept these semi-accurate times.

**Pull request closed without merging**